Tests/FoundationTests/FileManagerExtensionsTests.swift

```diff
@@ -80,6 +80,31 @@ final class FileManagerExtensionsTests: XCTestCase {
         } catch {}
     }
 
+    func testDecode() {
+        let point = CGPoint(x: 1, y: 2) //CGPoint conforms to Encodable
+        do {
+            let temporaryDirectoryURL = FileManager.default.temporaryDirectory
+            let temporaryFilename = ProcessInfo().globallyUniqueString
+            let temporaryFileURL = temporaryDirectoryURL.appendingPathComponent(temporaryFilename)
+
+            XCTAssertNoThrow(try JSONEncoder().encode(point))
+            let data1 = try JSONEncoder().encode(point)
+            XCTAssertTrue(FileManager.default.createFile(atPath: temporaryFileURL.path, contents: data1, attributes: nil))
+
+            XCTAssert(FileManager.default.fileExists(atPath: temporaryFileURL.path))
+
+            let data2 = FileManager.default.contents(atPath: temporaryFileURL.path)
+            XCTAssertNotNil(data2)
+
+            XCTAssertNoThrow(try FileManager.default.decode(as: CGPoint.self, from: temporaryFileURL))
```

Comment:
Again, since you're `catch`-ing you don't want to make this assertion. Instead you actually want to get the result back and test that it matches the original point that you saved.